### DISCUSSION TITLE
0x04 修改调用代码 ,定位不到代码，无法更改

### DISCUSSION THREAD

**Discussion by 0xsu** [NonceGeek/Move-Camp#239] [Q&A] [answered]
0x04 修改调用代码 ,定位不到代码，无法更改
<img width="627" alt="image" src="https://user-images.githubusercontent.com/101964290/212233402-2610b54c-b644-4372-8700-f6898e81f67f.png">

> https://mp.weixin.qq.com/s/qibzFk0IK3z_C5X6jfjA5Q

**Comment by leeduckgo** [comment]
代码源在这里。
用这个：

> https://github.com/NonceGeek/Web3-dApp-Camp/tree/main/move-dapp/my-counter/dapp/src

**Comment by leeduckgo** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/12784118/212243114-b1c02688-6e84-4fb4-a51a-1f6dfb2bb778.png)

**Reply by 0xsu** [reply]
报错
![Uploading 3FE4BCB8-BA4F-4DD8-ACED-F9250A4E04B6.png…]()

**Reply by 0xsu** [reply]
![image](https://user-images.githubusercontent.com/101964290/212894984-cfe9099a-eaee-4381-a513-13691643e3a3.png)

**Comment by 0xsu** [comment]
前端显示，但是调取合约显示链接错误
<img width="1305" alt="image" src="https://user-images.githubusercontent.com/101964290/213206903-6fe24999-0b40-4980-ac2b-38021a4f7a62.png">

**Reply by 0xsu** [reply]
修改了前端的项目，运行了yarn start，然后我钱包连接到我本地运行的 starcoin 网络上，然后就报错了

**Reply by 0xsu** [reply]
代码地址 https://github.com/0xsu/starcoin-test-dapp-react
